docs/essentials/exceptions/exceptions-best-practices.md

```diff
@@ -114,74 +76,17 @@ The following list contains guidelines for creating your own exceptions and when
   }
   ```
 
-  C++
-  ```
-  class FileRead
-  {
-  public:
-      void ReadAll(FileStream^ fileToRead)
-      {
-          // This if statement is optional
-          // as it is very unlikely that
-          // the stream would ever be null.
-          if (fileToRead == nullptr)
-          {
-              throw gcnew System::ArgumentNullException();
-          }
-
-          int b;
-
-          // Set the stream position to the beginning of the file.
-          fileToRead->Seek(0, SeekOrigin::Begin);
-
-          // Read each byte to the end of the file.
-          for (int i = 0; i < fileToRead->Length; i++)
-          {
-              b = fileToRead->ReadByte();
-              Console::Write(b.ToString());
-              // Or do something else with the byte.
-          }
-      }
-  };
-  ```
-
-  Visual Basic
-  ```
-  Class FileRead
-      Public Sub ReadAll(fileToRead As FileStream)
-          ' This if statement is optional
-          ' as it is very unlikely that
-          ' the stream would ever be null.
-          If fileToRead Is Nothing Then
-              Throw New System.ArgumentNullException()
-          End If
-
-          Dim b As Integer
-
-          ' Set the stream position to the beginning of the file.
-          fileToRead.Seek(0, SeekOrigin.Begin)
-
-          ' Read each byte to the end of the file.
-          For i As Integer = 0 To fileToRead.Length - 1
-              b = fileToRead.ReadByte()
-              Console.Write(b.ToString())
-              ' Or do something else with the byte.
-          Next i
-      End Sub
-  End Class
-  ```
-
 - Throw exceptions instead of returning an error code or HRESULT.
```

Comment:
We shouldn't call out HRESULT explicitly anymore since there are other error code types on other OSes.